packages/shorebird_cli/test/src/executables/patch_executable_test.dart

```diff
@@ -125,5 +131,46 @@ void main() {
         ),
       );
     });
+
+    group('when running on windows and process exits with code -1073741515', () {
+      setUp(() {
+        const stdout = 'uh oh';
+        const stderr = 'oops something went wrong';
+        when(() => patchProcessResult.exitCode).thenReturn(-1073741515);
+        when(() => patchProcessResult.stderr).thenReturn(stderr);
+        when(() => patchProcessResult.stdout).thenReturn(stdout);
+
+        when(
+          () => shorebirdProcess.run(
+            any(that: endsWith('patch')),
+            any(),
+            runInShell: any(named: 'runInShell'),
+          ),
+        ).thenAnswer((_) async => patchProcessResult);
+
+        when(() => platform.isWindows).thenReturn(true);
+      });
```

Comment:
We might prefer using `testOn` for the group instead of overriding `platform.isWindows`

Reply:
I have mixed feelings about that. Cause that would make our dev experience more difficult, because we would need to have a windows machine to develop this test and feature. And I just have a mac right now.

I feel that for small things like these, mocking the platform works better and testOn would makes more sense when we are making a more complex or when a feature "needs more features" from the platform?

Happy to change if you disagree though.

Reply:
Makes sense, it's ok to leave as-is